index.html

```diff
@@ -1,11 +1,125 @@
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
-    <h1>Testing</h1>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header>
+        <h1 id="title">Survey Form</h1>
+    </header>
+    <main class="container">
+        <p id="description">Let us know how we can improve freeCodeCamp</p>
+        <form id="survey-form" action="index.html" method="post">
+            <fieldset>
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="name-label">Name: *</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="text" id="name" placeholder="Enter your name" required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="email-label">* Email:</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="email" id="email" placeholder="Enter your Email" required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label id="number-label">* Age:</label>
+                    </div>
+                    <div class="right-row">
+                        <input class="input-field" type="number" id="number" placeholder="Age" min="15" max="60"
+                            required>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label>Which option best describes your current role?</label>
+                    </div>
+                    <div class="right-row">
+                        <select id="dropdown">
+                            <option value="1">Student</option>
+                            <option value="2">Full Time Job</option>
+                            <option value="3">Full Time Learner</option>
+                            <option value="4">Prefer not to say</option>
+                            <option value="5">Other</option>
+                        </select>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label>How likely is that you would recommend ThinkConcept to a friend?:</label>
+                    </div>
+                    <div class="right-row">
+                        <input type="radio" name="refer" value="Definitely">Definitely<br>
+                        <input type="radio" name="refer" value="Maybe">Maybe<br>
+                        <input type="radio" name="refer" value="Not sure">Not sure
+                    </div>
```

Comment:
# Radio form control

- Why don't you use semantic tags for the `.rowTab` element and its children instead of the generic tag `div` you used for grouping the label and the input controls? That would improve the overall structure of your code. Kindly see [how to build a form in HTML](https://developer.mozilla.org/en-US/docs/Learn/Forms/Your_first_form) for insights